config.yml

```diff
@@ -1997,6 +1979,18 @@ nodes:
 
           $1
           ^^
+  - name: OptionalKeywordParameterNode
+    fields:
+      - name: name
+        type: constant
+      - name: name_loc
+        type: location
+    comment: |
+      Represents an optional keyword parameter to a method, block, or lambda definition.
+
+          def a(b: )
+                ^^
+          end
```

Comment:
I would expect an "optional" keyword parameter to have a default value and required parameter not to have it.

Reply:
Right the names or so seem mixed up.

Ruby's terminology is `required` means has no default:
```
$ ruby -e 'p method(def m(k1:, k2: val); end).parameters'
[[:keyreq, :k1], [:key, :k2]]
```

Just like for positional required/optional arguments, the optional is the one with a default value.

Reply:
You're totally right, I got the names flipped. Thanks for catching!